src/content/en/2020/css.md

```diff
@@ -669,15 +715,19 @@ And if you are going for fanciful color names, why not define your own with CSS
 
-And we are still missing out. Despite being [implemented in Safari in 2016](https://webkit.org/blog/6682/improving-color-on-the-web/), the use of display-p3 color in Web pages is vanishingly small. Our crawl of the Web found only 29 mobile and 36 desktop pages (!) using it. (And more than half of those were syntax errors, mistakes, or attempts to use the never-implemented `color-mod()` function). We were curious why.
+And we are still missing out. Despite being [implemented in Safari in 2016](https://webkit.org/blog/6682/improving-color-on-the-web/), the use of display-p3 color in web pages is vanishingly small. Our crawl of the web found only 29 mobile and 36 desktop pages using it! (And more than half of those were syntax errors, mistakes, or attempts to use the never-implemented `color-mod()` function). We were curious why.
 
-Compatibility, right? You don’t want things to break? No. In the stylesheets we examined, we found solid use of fallback: with document order, the cascade, `@supports`, the `color-gamut` media query, all that good stuff. So in a style sheet we would see the color the designer wanted, expressed in display-p3, and also a fallback sRGB color. We computed the visible difference (a calculation called [ΔE2000](https://zschuessler.github.io/DeltaE/learn/)) between the desired and fallback color and this was typically quite modest. A small tweak. A careful exploration. In fact, 37.6% of the time, the color specified in display-p3 actually fell inside the range of colors (the gamut) that sRGB can manage.
+Compatibility, right? You don't want things to break? No. In the style sheets we examined, we found solid use of fallback: with document order, the cascade, `@supports`, the `color-gamut` media query, all that good stuff. So in a style sheet we would see the color the designer wanted, expressed in display-p3, and also a fallback sRGB color. We computed the visible difference (a calculation called [ΔE2000](https://zschuessler.github.io/DeltaE/learn/)) between the desired and fallback color and this was typically quite modest. A small tweak. A careful exploration. In fact, 37.6% of the time, the color specified in display-p3 actually fell inside the range of colors (the gamut) that sRGB can manage.
+
+{# TODO Authors—you're completely losing me with this :-) Not sure if there's a way to simplify it more or if this is just the nature of this topic and it's just more advanced, which is fine — the rest of the chapter so far is is more than readable for those less-advance amongst us! But still not sure what your point is here? They shouldn't have used display-p3 because sRGB was good enough? Or is it more than people are just cautiously experimenting at the moment so no real gains yet, but more to come (as covered by your next bit of writing after this table? #}
```

Comment:
@LeaVerou, @svgeesus, @rachelandrew - thoughts on this?

Reply:
Yes, it is a bit dense, but unpacking it more and giving more tutorial material would make it much longer.

> Or is it more than people are just cautiously experimenting at the moment so no real gains yet, but more to come

Yes, that is it.

I am actually considering making a whole blog post about just this one point. But that will take me some time.

Reply:
What do you think about adding that sentence to the end then?

```suggestion
Compatibility, right? You don't want things to break? No. In the style sheets we examined, we found solid use of fallback: with document order, the cascade, `@supports`, the `color-gamut` media query, all that good stuff. So in a style sheet we would see the color the designer wanted, expressed in display-p3, and also a fallback sRGB color. We computed the visible difference (a calculation called [ΔE2000](https://zschuessler.github.io/DeltaE/learn/)) between the desired and fallback color and this was typically quite modest. A small tweak. A careful exploration. In fact, 37.6% of the time, the color specified in display-p3 actually fell inside the range of colors (the gamut) that sRGB can manage. It seems people are just cautiously experimenting with thus at the moment rather than to get real gains, but more is surely to come so one to watch.
```